src/index.js

```diff
@@ -74,6 +74,15 @@ VirtualMachine.prototype.stopAll = function () {
 };
 
 /**
+ * Clear out current running project data.
+ */
+VirtualMachine.prototype.clear = function () {
+    this.runtime.dispose();
+    this.editingTarget = null;
+    this.emitTargetsUpdate();
```

Comment:
I like this way - a VM consumer could call `vm.clear()` and automatically get the target updates as expected.